tests/test_utils.py

```diff
@@ -365,6 +367,24 @@ def test_parse_datetime_err():
     assert parse_iso8824_date("D:20210408054711").tzinfo is None
 
 
+def test_format_datetime_to_pdf():
```

Comment:
For completeness, could be please add a roundtrip test besides the rendering check?